fandogh_cli/presenter/base_presenter.py

```diff
@@ -19,10 +19,18 @@ def table_renderer(data, **kwargs):
     headers = kwargs.get('headers')
     column_names = kwargs.get('columns')
     table = _create_table(headers)
-    for item in data:
+    for index, item in enumerate(data):
         row = []
         for cn in column_names:
-            if item.get(cn) is True:
+            if cn == 'restarts':
+                temp_count = 0
+                for pod in data[index].get('pods', ''):
```

Comment:
do this calculation on backend side, add a new field to the service to indicate how many times it's restarted